app/languages/definitions/Catalan.yml

```diff
@@ -0,0 +1,14 @@
+locale: ca-ES
+dictionaryFile: ca-ES-utf8.csv
+name: Català
+layout:
+  - [SPECIAL] # 0
+  - [PUNCTUATION_FR, ·] # 1
```

Comment:
Let's remove the dot and add the [l-dot](https://www.compart.com/en/unicode/U+0140) character on the 5-key. As I explained in #592, combining characters are potential trouble makers.

Reply:
Scratch that. As per the following comments, the middle dot will live separately.